apps/server/src/modules/board/controller/dto/element/update-element-content.body.params.ts

```diff
@@ -43,6 +43,7 @@ export class RichTextContentBody {
 }
 
 export class RichTextElementContentBody extends ElementContentBody {
+	// Validation?
```

Comment:
either start with TODO or remove comment
searching for TODO is common practice, and some IDEs will do that automatically

Reply:
Good catch